layers/synchronization2.cpp

```diff
@@ -1318,6 +1326,11 @@ VKAPI_ATTR VkResult VKAPI_CALL CreateRenderPass2(VkDevice device, const VkRender
                            VkRenderPass* pRenderPass) {
     auto device_data = GetDeviceData(device);
 
+    if (device_data->vtable.CreateRenderPass2 == nullptr) {
+        std::cerr << __func__ << " Device does not support CreateRenderPass2" << std::endl;
```

Comment:
Can we use the `LOG()` macro, mainly for Android Logcat printing (feel free to spice up the non-Android variation I quick threw there)

Reply:
good idea